src/components/SortSelector.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Select from '@material-ui/core/Select';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+
+export default function Subplaylist (props: {
+  setSort: Dispatch<SetStateAction<string>>
```

Comment:
Can we make this prop accept an unopinionated function callback; the current prop is very tied in to be implemented with a useState[1]-like value

Reply:
Not sure if this is what you mean, but have a look at the new code

Reply:
Looks good now :)